src/screens/Dashboard.js

```diff
@@ -6,10 +6,12 @@
  */
 
 import React from 'react';
+import ReactLoading from 'react-loading';
 import { connect } from "react-redux";
-import LineChartRealTime from '../components/LineChartRealTime';
-import AreaChartRealTime from '../components/AreaChartRealTime';
 import colors from '../index.scss';
+import helpers from '../utils/helpers';
+import economicsApi from '../api/economics';
```

Comment:
Starting a thread to discuss updating or not the economic metrics in realtime.

@pedroferreira1 actually the data updates every second, which I think we should change to 5 or 10 seconds.

I like the second suggestion. Maybe we should cache the mined tokens info on the server, as it only needs to be calculated when there's a new block. That would require a refactor on hathor core.

One other idea is just calling the economics api when the height of the best chain changes. That would keep this 100% realtime and prevent polling. But there would be a call to calculate the mined tokens everytime there's a new block, for each explorer that's open, so maybe caching it as I mentioned above would be better.